### ISSUE TITLE
Translation files are not installed

### ISSUE THREAD

**Issue by ryonakano** [torikulhabib/mindi#20]
I think translation files are not installed with the current configuration of Meson.

Add the following content in `meson.build`:

```
subdir('po')
```

And then add the following content in `po/meson.build` (new file):

```
i18n.gettext(meson.project_name(),
    args: [
        '--directory=' + meson.source_root(),
        '--from-code=UTF-8'
    ],
    preset: 'glib'
)
```

**Comment by NathanBnm** [comment]
@ryonakano maybe you should do a PR

**Comment by ryonakano** [comment]
Closing since this was fixed in 200abf9d4d2bc2bda64d47c1440fc835865bb80e, thanks  @torikulhabib!
